kindtest.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
Why this is needed ?

Reply:
I was trying to debug the docker registry issue. removed it.